Project-Starlord/Controllers/FollowerController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using Project_Starlord.Data;
+using Project_Starlord.Models;
+
+namespace Project_Starlord.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class FollowerController : ControllerBase
+    {
+        private readonly MyDbContext _context;
+
+        public FollowerController(MyDbContext context)
+        {
+            _context = context;
+        }
+
+        [Authorize]
+        [HttpPost]
+        public async Task<ActionResult<FollowerModel>> PostFollowerModel(FollowerModel follower)
+        {
+            _context.Followers.Add(follower);
+            await _context.SaveChangesAsync();
```

Comment:
I'm curious, what is the reason why this is async?

Reply:
Short answer performance. For a good explanation read this: https://stackoverflow.com/a/30043327